spec/rspec/rails/matchers/active_job_spec.rb

```diff
@@ -207,5 +207,13 @@ def to_global_id(options = {})
         }.to have_enqueued_job(hello_job).with(42).on_queue("low").at(date).exactly(2).times
       }.to raise_error(message)
     end
+
+    it "throws descriptive error when no test adapter set" do
+      ActiveJob::Base.queue_adapter = :inline
```

Comment:
normally I would do this with `ensure`, but rspec syntax doesn't allow it and not sure if wrapping whole code in `begin; end` block is fine.
Anyway I changed it